pkg/apiserver/user/sqlauth/sqlauth.go

```diff
@@ -60,3 +147,22 @@ func (a *Authenticator) Authenticate(f user.AuthenticateForm) (*utils.SessionUse
 		IsWriteable:  true,
 	}, nil
 }
+
+// grantRows examples:
+// - GRANT PROCESS,SHOW DATABASES,CONFIG ON *.* TO 'dashboardAdmin'@'%'
+// - GRANT SYSTEM_VARIABLES_ADMIN,RESTRICTED_VARIABLES_ADMIN,RESTRICTED_STATUS_ADMIN,RESTRICTED_TABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%'
+// - GRANT ALL PRIVILEGES ON *.* TO 'dashboardAdmin'@'%'
+func parseGrants(grantRows []string) []string {
```

Comment:
This parser can only parse global grants but not table grants. Prefer to indicate this feature in the name.